components/supervisor/pkg/serverapi/publicapi.go

```diff
@@ -169,28 +170,39 @@ func (s *Service) tryConnToPublicAPI() {
 	}
 }
 
+func (s *Service) observeConfigcatValue(ctx context.Context) {
+	ticker := time.NewTicker(time.Second * 10)
+	for {
+		select {
+		case <-ctx.Done():
+			ticker.Stop()
+			return
```

Comment:
At least for `ticker.Stop()` we need a `return` :grimacing: thanks to the lifecycle is the same with `supervisor`, we don’t have problem with it now